dummy_halo_generator.py

```diff
@@ -399,14 +399,18 @@ def get_random_halo(self, npart, has_neutrinos=False):
                 registry=reg,
             )
             data["PartType0"]["Velocities"] = vs[gas_mask]
+            xrays = np.zeros((Ngas, 3))
+            xrays[:, 0] = 10.0 ** (7.0 * np.random.random(Ngas))
+            xrays[:, 1] = 10.0 ** (7.0 * np.random.random(Ngas))
+            xrays[:, 2] = 10.0 ** (7.0 * np.random.random(Ngas))
             data["PartType0"]["XrayLuminosities"] = unyt.unyt_array(
-                10.0 ** (7.0 * np.random.random(Ngas)),
+                xrays,
                 dtype=np.float64,
                 units="snap_length**2*snap_mass/snap_time**3",
                 registry=reg,
             )
             data["PartType0"]["XrayPhotonLuminosities"] = unyt.unyt_array(
-                10.0 ** (70.0 * np.random.random(Ngas)),
+                xrays,
```

Comment:
Not sure what these random arrays were doing but they changed from 10^70 to 10^7.

Reply:
This generates "fake" haloes as unit tests for SOAP. This is for the x-ray in the fake snapshot. Changing 70 was a type and 70 is probably closer to the value in the snapshot, but as it is a unit test it doesn't really matter.